src/lib/components/inbox/conversations/__tests__/index-tests.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react"
+import "react-native"
+import * as renderer from "react-test-renderer"
+
+import Inbox from "../"
+
+it("looks correct when rendered", () => {
```

Comment:
Yeah but I think we should update Danger for that or do a Skip New Tests in the meantime because it's all over the app (e.g. [this implementation](https://github.com/artsy/emission/blob/master/src/lib/components/artist/articles/index.tsx) with [these tests](https://github.com/artsy/emission/blob/master/src/lib/components/artist/__tests__/articles-tests.tsx))